hyperdome_client/hyperdome_client.py

```diff
@@ -208,6 +204,13 @@ def send_message(self):
         Send the contents of the message box to the server to be forwarded to
         either counsel or guest.
         """
```

Comment:
this function can be flattened

Reply:
Flattened? What does that mean?

Reply:
less nesting, especially considering that the if statements are relatively complex expressions that aren't exceptionally clear as to their purpose. Breaking this out using the guard statement pattern or something else with an equivalent effect.